Removed call to arms. Added in lower down the page (#2418)
DFE-Digital/get-into-teaching-app: app/views/content/returning-to-teaching.md

#### Other things to consider

* attend an [online return to teaching event](/event-categories/online-q-as)

> * return to the classroom by [registering with a supply teaching agency](/urgent-call-for-qualified-teachers)

Reviewer: Thanks for changing this @tcannon3 - could you move it to be the last bullet point as well? Thanks!

Author: Oops sorry, thought I had. Will move it down now

> * return to the classroom on a temporary basis by joining a [supply teaching agency](https://www.gov.uk/government/publications/international-qualified-teacher-status-teachers-standards)

Reviewer: ```suggestion
* return to the classroom by [registering with a supply teaching agency](/urgent-call-for-qualified-teachers)
```

Reviewer: I wonder if we should put this as the last bullet - we would rather they considered returning in full rather than supply?

> * return to the classroom on a temporary basis by joining a [supply teaching agency](https://www.gov.uk/government/publications/international-qualified-teacher-status-teachers-standards)

Reviewer: ```suggestion
* return to the classroom by [registering with a supply teaching agency](/urgent-call-for-qualified-teachers)
```
